CharTokenizer.py is throwing System.InvalidOperationException during the nightly build

**Describe the bug**
In the extended tests, Ubuntu platform can't pass CharTokenizer.py. Below is the error message.

E           RuntimeError: Issue with
E             File '/builddir/src/python/nimbusml/examples/CharTokenizer.py'
E           --CMD
E           "/builddir/dependencies/Python3.6/bin/python" -u "/builddir/src/python/nimbusml/examples/CharTokenizer.py"
E           --ERR
E           Error: *** System.InvalidOperationException: 'Error downloading resource: Error trying to create directory : Path cannot be the empty string or all whitespace.
E           Parameter name: path.
E           Please fix your filesystem permissions, or try setting the 'MICROSOFTML_RESOURCE_PATH' environment variable to a writable folder
E             File "/builddir/src/python/nimbusml/examples/CharTokenizer.py", line 36, in <module>
E               features = pipe.fit_transform(data)
E             File "/builddir/dependencies/Python3.6/lib/python3.6/site-packages/nimbusml/internal/utils/utils.py", line 220, in wrapper
E               params = func(*args, **kwargs)
E             File "/builddir/dependencies/Python3.6/lib/python3.6/site-packages/nimbusml/pipeline.py", line 1186, in fit_transform
E               **params)
E             File "/builddir/dependencies/Python3.6/lib/python3.6/site-packages/nimbusml/internal/utils/utils.py", line 220, in wrapper
E               params = func(*args, **kwargs)
E             File "/builddir/dependencies/Python3.6/lib/python3.6/site-packages/nimbusml/pipeline.py", line 1152, in fit
E               raise e
E             File "/builddir/dependencies/Python3.6/lib/python3.6/site-packages/nimbusml/pipeline.py", line 1139, in fit
E               **params)
E             File "/builddir/dependencies/Python3.6/lib/python3.6/site-packages/nimbusml/internal/utils/entrypoints.py", line 293, in run
E               ret = self.idv_bridge(X, y, code, random_state, verbose, **params)
E             File "/builddir/dependencies/Python3.6/lib/python3.6/site-packages/nimbusml/internal/utils/entrypoints.py", line 463, in idv_bridge
E               output_modelfilename)
E             File "/builddir/dependencies/Python3.6/lib/python3.6/site-packages/nimbusml/internal/utils/entrypoints.py", line 334, in _try_call_bridge
E               raise e
E             File "/builddir/dependencies/Python3.6/lib/python3.6/site-packages/nimbusml/internal/utils/entrypoints.py", line 306, in _try_call_bridge
E               ret = px_call(call_parameters)
E           RuntimeError: Returned code is -1. Check the log for error messages.

--- a/src/python/tests_extended/test_docs_example.py
+++ b/src/python/tests_extended/test_docs_example.py
@@ -65,6 +65,7 @@ class TestDocsExamples(unittest.TestCase):
                     'SymSgdBinaryClassifier.py',
                     'SymSgdBinaryClassifier_infert_df.py',
                     # MICROSOFTML_RESOURCE_PATH needs to be setup on linux
+                    'CharTokenizer.py',
                     'WordEmbedding.py',
                     'WordEmbedding_df.py',
                     'NaiveBayesClassifier_df.py'

